salt/modules/win_pkg.py

```diff
@@ -1243,7 +1243,7 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
 
             # Run Scheduled Task
             # Special handling for installing salt
-            if pkg_name in ['salt-minion', 'salt-minion-py3']:
+            if re.search(r'salt[\s-_]*minion', pkg_name, flags=re.IGNORECASE+re.UNICODE) is not None:
```

Comment:
`r'salt[\s-_]*minion' `  needs to be `r'salt[\s_-]*minion'` or `r'salt[\s_.-]*minion'` (with . if you want salt.minion)
As like `[a-z]` unless `-` is last it means a range.